autogluon/inference/docker/1.1/py3/Dockerfile.cpu

```diff
@@ -35,7 +35,7 @@ RUN pip install --no-cache-dir -U --trusted-host pypi.org --trusted-host files.p
  && pip install --no-cache-dir -U mmdet>=3.0.0 \
  && pip install --no-cache-dir -U "mmengine" \
 # Update transformers to secure versions that fixes CVE-2023-7018
- && pip install --no-cache-dir -U setuptools>=65.5.1 \  
+ && pip install --no-cache-dir setuptools==69.5.1 \  
```

Comment:
Could you please provide a brief justification as comment for why we need to anchor the setuptools to a particular version?

Reply:
addressed comments